task/conftest-clair/0.1/conftest-clair.yaml

```diff
@@ -33,8 +33,16 @@ spec:
                    timestamp: $date,
                    namespace,
                    successes,
-                   failures: (.failures // [])|map(.metadata.details.name) | unique
+                   failures: (.failures | length)
                  }' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json || true)
+
+        # Log out the failing runs
+        if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+        then
+          echo "The clair-vulnerabilities test has failed with the following issues::"
+          echo $(jq '.[].failures // []|map(.metadata.details.name) | unique' $(workspaces.conftest-ws.path)/clair-vulnerabilities.json)
+        fi
+
```

Comment:
Don't you want to create a script in `quay.io/redhat-appstudio/hacbs-test:latest` ? So that it would not be additional step in each test?

Reply:
Yes, using a script is better to handle these similar steps. I am thinking two points:
1. Add script to handle the step of logging out the failing runs or to judge $HACBS_TEST_OUTPUT and then logging out the failing runs?
2. We have two types of failing runs in json file to parse: `'.[].failures // []|map(.metadata.details.name)` and `.runs[].results // []|map(.message.text)`, should we create one script with some parameters or two scripts without less parameters?

Reply:
I think that creating one script with different parameters for different output formats (conftest json, sarif json etc.) would make the most sense. It could also log out the failures so they're visible in the TaskRun log.
I don't think that this falls under the scope of this PR, we will create a follow-up story to implement the script and unify the test output/result handling.

Reply:
[HACBS-1386](https://issues.redhat.com/browse/HACBS-1386) is filed to track the work. Thank you all!